src/js/controllers/ctrl-create.js

```diff
@@ -246,14 +245,14 @@ app.controller('createCtrl', function(
 							case 'setHeight': // the height of the creator has changed
 								return setHeight(msg.data[0])
 							case 'alert':
-								return _alert(msg.data)
+								return _alert(msg.data.msg, msg.data.title, msg.data.fatal)
 							default:
 								return console.warn(`Unknown message from creator: ${msg.type}`)
 						}
-				}				
+				}
 			}
 
-			_alert(`Error, cross domain restricted for ${origin}`)
+			console.warn(`Unknown message from creator: ${origin}`)
```

Comment:
Change: unknown post messages caught in the creator core will no longer display an alert